javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalCost: 34,
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    deliveredDate: 'Aug 8, 2020',
+    returnWindow: 'Sep 7, 2020',
+    itemList: [{
+      name: 'JavaScript for impatient programmers',
+      type: 'book',
+      author: 'Raushmayer, Dr. Axel',
+      cost: 31.55
+    }]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    totalCost: 44.53,
+    shipTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    deliveredDate: 'Jul 20, 2020',
+    returnWindow: 'Aug 19, 2020',
+    itemList: [{
+      name: 'The Timeless Way of Building',
+      type: 'book',
+      author: 'Alexander, Christopher',
+      cost: 41.33
```

Comment:
Great way to organize the data! I am still missing the image though. You don't need a real url or anything, just something to show that the image is being stored on each item.